src/classes/ALLO_Allocations_TEST.cls

```diff
@@ -124,6 +124,71 @@ private with sharing class ALLO_Allocations_TEST {
         system.assertEquals(8, queryOpps[0].Amount, 'The opportunity amount should remain unchanged after the exception is caught.');
     }
 
+
+    /*******************************************************************************************************
+    * @description When changing the amount of a Payment with Allocations: 
+    * Increasing the amount will adjust the percentage allocations.
+    * Decreasing the amount so that the total allocation amount is below the payment amount will adjust 
+    * the percentage allocations.
+    * Decreasing the amount so that the total allocation amount is greater than the payment amount, an 
+    * error is thrown and the payment update is blocked.
+    ********************************************************************************************************/
+    static testMethod void pmtAmountChange() {
+        if (strTestOnly != '*' && strTestOnly != 'pmtAmountChange') return;
+        setupSettings(new Allocations_Settings__c(Payment_Allocations_Enabled__c = true));
+
+        Account acc = new Account(Name='foo');
```

Comment:
@pedrodalcol I know this isn't exactly clearly documented yet, but in general we're trying to avoid the use of simple and repeated `Name` values in Unit Tests, such as "foo". It's part of the larger issue of parallel unit tests. Instead, would you mind using something more unique to each test method such as `Name='Account-pmtAmountChange'`?

Reply:
Good one, @force2b. In HEDA we have a utility test class that generates records with unique values for indexed fields: https://github.com/SalesforceFoundation/HEDAP/blob/master/src/classes/UTIL_UnitTestData_TEST.cls. Just an idea, in case you want to use something like that in NPSP.

Reply:
Thanks @ceiroa. I started building out something like that when I began the parallel unit test experiments. When we get the point of re-starting that work, we'll reach out to talk about how you're handling the HEDA testing framework.